src/Snowcap/Emarsys/Client.php

```diff
@@ -250,17 +250,24 @@ public function createContact(array $data)
      * Updates one or more contacts/recipients, identified by an external ID.
      *
      * @param array $data
+     * @param bool $createIfNotExists When enabled, if the contact does not exist in the database, it is created
+     * automatically.
      * @return Response
+     * @link http://documentation.emarsys.com/resource/developers/endpoints/contacts/update-contacts/
      */
-    public function updateContact(array $data)
+    public function updateContact(array $data, $createIfNotExists = false)
```

Comment:
Let's avoid optional parameters. Implementing a new method like `updateContactAndCreateIfNotExists` would be my preferred way.